DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: ospsuite.plots
 Type: Package
 Title: Library for standardized graphs
-Version: 1.0.0
+Version: 1.0.0.9000
```

Comment:
As long as in the development, the 9000 build number will be automatically increased with every push into the main branch.